lib/fork.js

```diff
@@ -18,13 +18,29 @@ module.exports = function (args) {
 
 	var ps = childProcess.fork(babel, args, options);
 
+	var killed = false;
+	var killedPromise = new Promise(function (resolve) {
+		ps.on('exit', function (code) {
+			killed = true;
+			resolve(code);
+		});
+	});
+
+	function kill() {
+		if (!killed) {
+			ps.kill();
+		}
+		return killedPromise;
+	}
+
 	var promise = new Promise(function (resolve, reject) {
 		ps.on('results', function (results) {
+			results.kill = kill;
```

Comment:
Then we better store references to forks, rather than injecting a function in result object.

> Not sure how this would create "weird stuff" in test results.

Function in an object with test results is weird stuff.